pycloudlib/cloud.py

```diff
@@ -16,6 +17,15 @@
 from pycloudlib.util import get_timestamped_tag, validate_tag
 
 
+@enum.unique
+class ImageType(enum.Enum):
+    """Allowed image types when launching cloud images."""
+
+    GENERIC = "generic"
+    PRO = "PRO"
+    PRO_FIPS = "FIPS PRO"
```

Comment:
Good catch on reversed name, but the uppercase is just because we usually refer to `PRO` instances using all letters in uppercase. And since we will surface that name on the Exceptions, I have kept that it just for consistency. But I can update that if necessary